lib/mix/lib/mix/project.ex

```diff
@@ -67,9 +67,15 @@ defmodule Mix.Project do
     * `:preferred_cli_env` - a keyword list of `{task, env}` tuples where `task`
       is the task name as an atom (for example, `:"deps.get"`) and `env` is the
       preferred environment (for example, `:test`). This option overrides what
-      specified by the tasks with the `@preferred_cli_env` attribute (see the
+      is specified by the tasks with the `@preferred_cli_env` attribute (see the
       docs for `Mix.Task`). Defaults to `[]`.
 
+    * `:preferred_cli_target` - a keyword list of `{task, target}` tuples where
+      `task` is the task name as an atom (for example, `:"test"`) and `target`
```

Comment:
Those quotes are unnecessary, and newer Elixir versions warn in such cases (and the formatter removes them) 🙂 

```suggestion
      `task` is the task name as an atom (for example, `:test`) and `target`
```

Reply:
AH good catch!